### PR TITLE
Fix: TC build config has been renamed to

### PR DESCRIPTION
**Pull request by TBonnin**
Fix: TC build config has been renamed from 'dotcom_master' to 'dotcom_frontend'

This patch reflects this renaming

### CODE REVIEW

File: admin/app/controllers/RadiatorController.scala [hunk 1]
```diff
@@ -49,7 +49,7 @@ class RadiatorController(wsClient: WSClient)(implicit context: ApplicationContex
     }
 
     for {
-      ciBuilds <- mostRecentBuildForProjects("dotcom_master", "dotcom_ampValidation")
+      ciBuilds <- mostRecentBuildForProjects("dotcom_frontend", "dotcom_ampValidation")
```
Comment (jfsoul) [inline admin/app/controllers/RadiatorController.scala]: Do we mind that this is going to fetch more than just master builds now?

**Comment by MatthewJWalls** [comment]
PR build results: 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-15508/screenshots/mobile.png 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-15508/screenshots/wide.png 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-15508/screenshots/tablet.png 
* https://s3-eu-west-1.amazonaws.com/prbuilds/PR-15508/screenshots/desktop.png 

 -automated message

File: admin/app/controllers/RadiatorController.scala (hunk 1 above)
Comment (TBonnin) [inline admin/app/controllers/RadiatorController.scala]: We actually do mind. Let me revise this PR then

File: admin/app/model/deploys/Teamcity.scala [hunk 2]
```diff
@@ -79,10 +81,10 @@ class TeamcityService(httpClient: HttpLike) extends ExecutionContexts {
       }
   }
 
-  def getBuilds(project: String, count: Int = 10): Future[Seq[TeamCityBuild]] = {
+  def getBuilds(project: String, branch: String = "master", count: Int = 10): Future[Seq[TeamCityBuild]] = {
     GET[TeamCityBuilds](
       path = "builds",
-      queryString = Map("locator" -> s"buildType:(id:$project),count:$count") + ("fields" -> s"build($buildFields)")
+      queryString = Map("locator" -> s"buildType:(id:$project),branch:$branch,count:$count") + ("fields" -> s"build($buildFields)")
```
Comment (TBonnin) [inline admin/app/model/deploys/Teamcity.scala]: It looks like if you don't specify a branch value, Teamcity is using the default branch (which is master in our case). So in theory those changes above are not necessary but they make things explicit 💃

File: admin/app/controllers/RadiatorController.scala [hunk 3]
```diff
@@ -43,13 +43,13 @@ class RadiatorController(wsClient: WSClient)(implicit context: ApplicationContex
     def mostRecentBuildForProjects(projects: String*): Future[Seq[TeamCityBuild]] = {
       Future.sequence(projects.map { project =>
         teamcityService
-          .getBuilds(project, 1)
+          .getBuilds(project, count=1)
```
Comment (jfsoul) [inline admin/app/controllers/RadiatorController.scala]: Nitpick: spacing around `=`

**Review by jfsoul — APPROVED** [review]

**Pull request merged**